docs/code-search/code-navigation/envvars.mdx

```diff
@@ -1,12 +1,12 @@
-# Code navigation environment variables
+# Code Navigation Environment Variables
 
-All of the following environment variables are prefixed with `PRECISE_CODE_INTEL`.
+<p className="subtitle">These docs list supported environment variables for Code Navigation. All of these environment variables are prefixed with `PRECISE_CODE_INTEL`.</p>
```

Comment:
Nit: 'Code navigation' is a feature of the 'Code Search' product. Per the handbook, 'Code navigation' should have a lowercase 'N'.

https://handbook.sourcegraph.com/company-info-and-process/communication/content_guidelines/style_and_mechanics/#feature-names